common/build-ios.sh

```diff
@@ -63,11 +63,12 @@ build_xcframework() {
   fi
 }
 
-cargo build --lib --release --target x86_64-apple-ios
-cargo build --lib --release --target aarch64-apple-ios-sim
-cargo build --lib --release --target aarch64-apple-ios
-
 basename=ferrostar
+
+cargo build -p $basename --lib --release --target x86_64-apple-ios
+cargo build -p $basename --lib --release --target aarch64-apple-ios-sim
+cargo build -p $basename --lib --release --target aarch64-apple-ios
```

Comment:
Minor improvement: we don't need to build `uniffi-bindgen` for any other architectures.